apis/r/man/ConfigList.Rd

```diff
@@ -10,7 +10,7 @@ Essentially, serves as a nested map where the inner map is a
 \code{\{<param>: \link[tiledbsoma:ScalarMap]{\{<key>: <value>\}}\}}
 }
 \section{Super class}{
-\code{\link[tiledbsoma:MappingBase]{tiledbsoma::MappingBase}} -> \code{ConfigList}
+\code{tiledbsoma::MappingBase} -> \code{ConfigList}
```

Comment:
I am pretty sure we playing commit ping-pong here:

```
43d09be3c (Dirk Eddelbuettel 2023-06-21 21:22:59 -0500  13) \code{\link[tiledbsoma:MappingBase]{tiledbsoma::MappingBase}} -> \code{ConfigList}
```

I am using roxygen2 7.2.3.  What else could be different?  I am not setting an environment variables or magic.

Reply:
Roxygen2 results in different behavior depending on how it's run (eg. RStudio/R console vs `Rscript -e`). I find running it with `Rscript -e` works best, especially with regards to auto-linking docs

Reply:
Same, albeit via `r`.  I always use my `roxy.r` wrapper, with option `-f` for full:

```r
## loop over all argument, with fallback of the current directory, and
## call roxygenize() on the given directory with roclets="rd" set
sapply(ifelse(length(argv) > 0, argv, "."),
       FUN = roxygenize,
       roclets = if (tolower(opt$roclets) == "null") NULL else strsplit(opt$roclets,",")[[1]])
```

Do we know / can we know what RStudio, or VSCode, or ... do differently?  Can we settle on a harmonizing step that unifies this?   I have not strong view in the matter and mostly deploy `roxygen2` so that it stays out of my hair (i.e. my personal default use it version 6.0.1 with just `namespace,rd` as roclets as strictly no fancy pants).

Reply:
I reverted since my changes don't require any edits to the docs.